lessons/Early-Math/associative-property/langs/en-us/index.adoc

```diff
@@ -0,0 +1,123 @@
+
+Students will apply this understanding in the next activity.
+
+[.lesson-instruction]
+- On @printable-exercise{assoc-prop-table.adoc}, apply the Associative Property to create @vocab{equivalent} Circles of Evaluation. Pay attention to the operators!
+- Now try @printable-exercise{true-or-false.adoc}. If the equation is true, explain how you know. If it is false, explain what you would need to change to make it true.
+
+Follow up to confirm understanding of the activities, paying specific attention to expressions with multiple operations. In these instances, students _can_ apply the Associative Property - but only on the parts of the expression that feature only addition or only multiplication.
+
+=== Synthesize
+
+- For which operations does the Associative Property hold?
+** _The Associative Property holds for multiplication and addition, but not division or subtraction._
+- Can you ever apply the Associative Property on an expression that includes division or subtraction?
+** _Yes. If the expression *also* includes addition or multiplication, there might be an opportunity to change the groupings without changing the value of the expression._
+- When applying the Associative Property, did you ever change the order of the numbers?
+** _Yes, students will change the numbers' order throughout the lesson._
+- When applying the Associative Property, did you ever change the architecture of the circles? (For instance, did you ever change “three nested circles” to “two-circles-in-a-circle”?) When was that necessary? Why?
+** _Yes, sometimes to change the groupings, we also changed the structure of the expression._
+
+== The Associative Property and Mental Math @duration{25 minutes}
+
+=== Overview
+Students continue to explore how the Associative Property and Circles of Evaluation make mental computation simpler.
+
+=== Launch
+
+[.lesson-instruction]
+- Think of an addition or multiplication problem that _appears_ to be very challenging, but is _much_ easier to solve after applying the Associative Property. Write it down on a piece of paper. Try to be creative!
+- Trade papers with a partner. How do your problems compare?
+- Turn your paper in to your teacher.
+
+We encourage you to review students' submissions, and write a few on the board to discuss as a class. We want to sharpen students' eyes and help them develop the ability to spot instances when they might apply the Associative Property in any context.
+
+=== Investigate
+
+[.lesson-instruction]
+- Complete @printable-exercise{which-circles-make-the-math-easier.adoc}. Examine each Circle of Evaluation and put a check mark by the one that results in a simpler computation. Then, evaluate the Circle.
+- Now, try @printable-exercise{props-make-comp-easier.adoc} to apply the Associative Property to expressions with more numbers and varying operators.
+- Try @printable-exercise{props-make-comp-easier-challenge.adoc} to apply the Associative Property to expressions with fractions and decimals.
```

Comment:
I see that you have titled the asiidoc challenge... but I would want to call out much more clearly in the lesson plan that this is a leap that not all teachers should make!

eg
If your students are comfortable with computation involving fractions and decimals, challenge them to apply what they've learned about the Associative Property to @...